bin/hammer

```diff
@@ -52,12 +52,16 @@ end
 
 CFG_PATH_LOCAL = ['./config/']
 
+HammerCLI::Settings.load_from_paths CFG_PATH_LOCAL
+
 if preparser.config
-  CFG_PATH_LOCAL.unshift preparser.config
+  if File.file? preparser.config
+    HammerCLI::Settings.load_from_file preparser.config
```

Comment:
That's not completely true. Bash won't expand it when you pass the parameter as a string. E.g.
```
hammer -c '~/some_config.yml'
```
I think this should be supported usage.